base/sparse/linalg.jl

```diff
@@ -941,3 +941,26 @@ end
 chol(A::SparseMatrixCSC) = error("Use cholfact() instead of chol() for sparse matrices.")
 lu(A::SparseMatrixCSC) = error("Use lufact() instead of lu() for sparse matrices.")
 eig(A::SparseMatrixCSC) = error("Use eigs() instead of eig() for sparse matrices.")
+
+function spcov(X::SparseMatrixCSC{Tv1,Ti1} where {Tv1,Ti1})
```

Comment:
Yes, as Fredrik points out, this should be `cov` since we want it to be a method of `Base.cov`. That way if someone calls `cov` on a matrix that happens to be sparse, they get the better-performing version without having to remember to call `spcov`.